spec/infinispan_ssl_spec.js

```diff
@@ -97,6 +133,16 @@ describe('Infinispan TLS/SSL client', function() {
     }
   }
 
+  function sslSniTrusted1() {
```

Comment:
I have renamed all necessary functions. This test tests that if server has 2 trusted sni-s, then both can be used for the authentication.